src/pybind/cephfs/cephfs.pyx

```diff
@@ -21,8 +21,22 @@ if sys.version_info[0] < 3:
 else:
     str_type = str
 
-cdef int AT_SYMLINK_NOFOLLOW = 0x100
-cdef int CEPH_STATX_BASIC_STATS = 0x7ffu
+AT_NO_ATTR_SYNC = 0x4000
+AT_SYMLINK_NOFOLLOW = 0x100
+CEPH_STATX_BASIC_STATS = 0x7ff
```

Comment:
Do not use cdef definitions because you need to call variables directly via test_cephfs.py